packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx

```diff
@@ -57,6 +57,16 @@ describe("GIVEN a Tooltip", () => {
     });
   });
 
+  describe.only("WHEN disabled", () => {
+    it("should not render when form field context disabled is undefined", () => {
+      cy.mount(
+        <FormField>
+          <Default disabled />
```

Comment:
I believe this is a false positive, the Tooltip hasn't been opened yet so won't be visible.

Reply:
ah true, will use an open one